tests/runtime-trace-tests/cases/for_loop.ts

```diff
@@ -0,0 +1,3 @@
+for (let i = 0; i < 5; i++) {
+    console.log(i)
+}
```

Comment:
@riknoll , here's a test case. There's no "actual" and "expected" explicitly written here. Instead we run this .ts program with node.js and that stdout becomes the "expected", then we decompile it python and run the output with python3 and that result is the "actual" and if they disagree, we know there's probably a ts2py issue. See the readme.md for more info. 

We also compare the node.js trace with the sts trace, and we (will soon) compare traces before and after py2ts.

Reply:
The point of these tests is to check that our runtime semantics in static typescript/python line up with tsc+node.js/python3